.github/workflows/linting.yaml

```diff
@@ -40,7 +40,30 @@ jobs:
         timeout: 10
 
         # Exclude these patterns from the checker
-        exclude_patterns: supercomputing.org,https://pace.gatech.edu,https://www.linkedin.com,jobs.colorado.edu,zoom.us,danielskatz.org,usrse.github.io.wiki.git,ornl.gov,jobs.bnl.gov,https://www.rd-alliance.org/,https://uwhires.admin.washington.edu/,https://careers.umich.edu/,https://twitter.com/us_rse,https://twitter.com/us_rse/status/1447622175133945860,https://twitter.com/iancosden/status/1122937311644323841
+        exclude_patterns: "supercomputing.org,\
+          https://pace.gatech.edu,\
+          https://www.linkedin.com,\
+          jobs.colorado.edu,zoom.us,\
+          danielskatz.org,\
+          usrse.github.io.wiki.git,\
+          ornl.gov,\
+          jobs.bnl.gov,\
+          https://www.rd-alliance.org/,\
+          https://uwhires.admin.washington.edu/,\
+          https://careers.umich.edu/,\
+          https://twitter.com/us_rse,\
+          https://twitter.com/us_rse/status/1447622175133945860,\
+          https://twitter.com/iancosden/status/1122937311644323841,\
+          https://bitcon.blacksintechnology.net/"
 
         # Exclude these files from the checker
-        exclude_files: README.md,docs/events.md,docs/tests_ci.md,docs/local_previews.md,SocialNetworks.yml,map.yml,_config.yml,tests/test_,.github/workflows,_posts/newsletters/
+        exclude_files: "README.md,\
+          docs/events.md,\
+          docs/tests_ci.md,\
+          docs/local_previews.md,\
+          SocialNetworks.yml,\
+          map.yml,\
+          _config.yml,\
+          tests/test_,\
+          .github/workflows,\
+          _posts/newsletters/"
```

Comment:
Making note in case others would like to do this another way. This is definitely better than one really long comma-separated string.